packages/vx-text/package.json

```diff
@@ -29,11 +29,10 @@
   "homepage": "https://github.com/hshoff/vx#readme",
   "dependencies": {
     "@types/classnames": "^2.2.9",
-    "@types/lodash": "^4.14.146",
+    "@types/lodash.memoize": "^4.1.6",
     "@types/react": "*",
     "classnames": "^2.2.5",
-    "lodash": "^4.17.15",
-    "prop-types": "^15.7.2",
+    "lodash.memoize": "^4.1.2",
```

Comment:
I've gone through and done this before but had forgot this was intentional, see #66

Reply:
oh ok sure. will revert.